Source/Core/Core/ConfigManager.cpp

```diff
@@ -249,6 +249,8 @@ void SConfig::SaveCoreSettings(IniFile& ini)
   core->Set("OverrideGCLang", bOverrideGCLanguage);
   core->Set("DPL2Decoder", bDPL2Decoder);
   core->Set("Latency", iLatency);
+  core->Set("AudioStretch", m_audio_stretch);
```

Comment:
Ishiiruka uses the variable "TimeStretching," it might be nice to match (if users share configuration between the two programs)